workshops/networking/1.6-roles/README.md

```diff
@@ -0,0 +1,188 @@
+# Exercise 1.6 - Roles: Making your playbooks reusable
+
+While it is possible to write a playbook in one file as we’ve done throughout this workshop, eventually you’ll want to reuse files and start to organize things.
+
+Ansible Roles is the way we do this. When you create a role, you deconstruct your playbook into parts and those parts sit in a directory structure. "Wha?? You mean that seemingly useless [best practice](http://docs.ansible.com/ansible/playbooks_best_practices.html) you mentioned in exercise 1.2?". Yep, that one.
```

Comment:
>Also what do you mean about that "seemingly useless" best practices doc?

@tima sorry this is recycled content from http://ansible.redhatgov.io/standard/welcome/index.html

I didn't write it, just reformatted it into markdown.  I assume it was a joke because the best practices covered roles well before the student knew what they were.

1.1->1.5 have been thoroughly scrubbed for content like this, but 1.6 was still in beta... we were not sure if there would be enough time given the class.  I will go ahead and fix 1.6 as well as some things have slipped through.  I have gone ahead and fixed this with a commit here: https://github.com/network-automation/lightbulb/commit/5bc6ea6f5d98c80df2e75bad4224e181a368b12a